frontend/src/features/comparisons/ComparisonList.tsx

```diff
@@ -67,10 +58,18 @@ const Comparisons = ({
 }: {
   comparisons: Comparison[] | undefined;
 }) => {
-  const classes = useStyles();
+  const theme = useTheme();
 
   return (
-    <Container className={classes.content}>
+    <Container
+      sx={{
+        padding: theme.spacing(3),
+        [theme.breakpoints.down('sm')]: {
+          padding: theme.spacing(0),
+        },
+        maxWidth: 840,
```

Comment:
The `!important` is needed here to match the previous layout. Without it, the container takes up to 1200px in full screen.

```suggestion
        maxWidth: '840px !important',
```

Reply:
@GresilleSiffle Updated the _ComparisonList_ component as you suggested.